### DISCUSSION TITLE
Date of last connection for [Not 24/7] streams

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#54] [Ideas]
To reduce potential complaints about fix dead or missing channels, this proposal may help.

Taking advantage of the fact that the bot performs daily updates, a last online check date could be added for [Not 24/7] channels. This date is not only useful for statistics, but also to identify possible completely offline channels. The date of last conexion could be consulted in json.

If the bot fails to connect to the channel after a period of time (it could be a week), the stream will automatically change its tag as [Offline].

Updated from iptv-org/iptv-org.github.io#150

Personal note: As the data is displayed in JSON, some developer will be able to sort from the last oldest connection and then easily "follow" the channels that would be permanently "offline".

**Comment by sguinetti** [comment]
I think the day of last connection should be in the API. Or am I wrong?

**Reply by BellezaEmporium** [reply]
Kind of relates to my thread here : https://github.com/iptv-org/iptv/discussions/7335

**Comment by freearhey** [comment]
#8003
